kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java

```diff
@@ -63,6 +65,9 @@
   public static final String REGION_CONFIG = "s3.region";
   public static final String REGION_DEFAULT = Regions.DEFAULT_REGION.getName();
 
+  public static final String ACL_CANNED_CONFIG = "s3.acl.canned";
+  public static final String ACL_CANNED_DEFAULT = CannedAccessControlList.Private.toString();
```

Comment:
Handling the default as null adds significant additional complexity. I'm open to continuing down that path if you think the additional complexity is worth it, @kkonstantine.